### ISSUE TITLE
Enhance documentation

### ISSUE THREAD

**Issue by arnuschky** [kjur/jsjws#9]
This looks like it's exactly what I need, but there seems to be almost no documentation. There's the API docs, sure, but nothing else. Readme is barebone, and the tutorials link directs to an empty wiki. I tried reading the code on the site, but either I have version or dependency issues.

Some questions:
 * What dependencies are required?
 * What is the relation between jsjws and jsrasign? jsjws seems to depend on jsrasign, but why it then the former included in the install of the latter?
 * Basic working examples would be nice, for a recent version. The stuff I got from the demos page seems outdated, or at least didn't work for me.
 * Basically a how-to is missing
 * What's the difference of readSafeJSONString wrt JSON.parse?

**Comment by kjur** [comment]
The 'jsjws' have been merged into latest 'jsrsasign' 4.8.0 released today. Tutorial wiki pages s are added.
https://github.com/kjur/jsrsasign/wiki
Dependency issue is described in the wiki.
